modules/custom/social_graphql/tests/src/Kernel/GraphQLTestBase.php

```diff
@@ -42,7 +42,16 @@ abstract class GraphQLTestBase extends KernelTestBase {
   use UserCreationTrait;
 
   /**
-   * {@inheritdoc}
+   * Modules to enable.
+   *
+   * The test runner will merge the $modules lists from this class, the class
+   * it extends, and so on up the class hierarchy. It is not necessary to
+   * include modules in your list that a parent class has already declared.
+   *
+   * @see \Drupal\Tests\KernelTestBase::enableModules()
+   * @see \Drupal\Tests\KernelTestBase::bootKernel()
+   *
+   * @var string[]
```

Comment:
Here we overwrote the `inheritdoc` because Drupal core actually just has `@var array` (this would make a good issue to file against Drupal to improve).

We could maybe combine `{@inheritdoc}` with `@var string[]` but I haven't tried that.

Reply:
Please note that `@var` should be above `@see` in the above example (apparently Drupal Core also didn't do this correctly). This was fixed in a new commit.